src/pbs.py

```diff
@@ -118,19 +115,13 @@ def execute(self):
         script_content = "\n".join(self.pbs_script)
         pbs_file = os.path.join(self._job_dir, "{:04d}.sh".format(self._job_count))
 
-        pbs_file_pom = os.path.join("/storage/liberec1-tul/home/martin_spetlik",
-                                    "{:04d}.sh".format(self._job_count))
         self._job_count += 1
-        with open(pbs_file_pom, "w") as file_writer:
+        with open(pbs_file, "w") as file_writer:
```

Comment:
It seems as workaround for problems with astra share, not necessary anymore. I propose just write into pbs_file directly.